solr/solrj/src/java/org/apache/solr/client/solrj/impl/BaseHttpClusterStateProvider.java

```diff
@@ -138,8 +138,7 @@ private ClusterState fetchClusterState(SolrClient client, String collection, Map
     Set<String> liveNodes = new HashSet((List<String>)(cluster.get("live_nodes")));
     this.liveNodes = liveNodes;
     liveNodesTimestamp = System.nanoTime();
-    //TODO SOLR-11877 we don't know the znode path; CLUSTER_STATE is probably wrong leading to bad stateFormat
```

Comment:
Remember to close SOLR-11877 after this

Reply:
Yes, it's mentioned in the PR description.